### ISSUE TITLE
Webpack 4 support

### ISSUE THREAD

**Issue by thaoms** [statianzo/webpack-livereload-plugin#40]
This plugin breaks in Webpack 4.

`````
(node:31933) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
(node:31933) DeprecationWarning: Tapable.apply is deprecated. Call apply on the plugin directly instead
Live Reload listening on port 35729

/Users/thomas/Desktop/websites/chilli-start-2/node_modules/tapable/lib/Tapable.js:63
		throw new Error(`Plugin could not be registered at '${name}'. Hook was not found.\n` +
		^

Error: Plugin could not be registered at 'html-webpack-plugin-before-html-generation'. Hook was not found.
BREAKING CHANGE: There need to exist a hook at 'this.hooks'. To create a compatiblity layer for this hook, hook into 'this._pluginCompat'.

`````

**Comment by ilkkao** [comment]
`html-webpack-plugin-before-html-generation` error doesn't seem to be related to this plugin. I'm only seeing the first warning when using this plugin alone. Fixed it here: https://github.com/statianzo/webpack-livereload-plugin/pull/42

**Comment by statianzo** [comment]
Merged the fix in #42. Thanks.
